src/colvar.cpp

```diff
@@ -439,6 +439,21 @@ int colvar::init_custom_function(std::string const &conf)
 
 int colvar::init_custom_function(std::string const &conf)
```

Comment:
Right now the only call to this function is under an #ifdef LEPTON already, so I don't think this version of the function is ever called. But the code looks good to me, if it was to be called.

Reply:
This version would be called if Lepton is turned off in LAMMPS, manually via `COLVARS_LEPTON=no` or automatically by `DISABLE_CXX11_REQUIREMENT`.  It is possible, though rare, that someone builds Colvars with C++11 but with Lepton explicitly disabled.  Hence the second `#ifdef`, which is definitely redundant in most cases by far.